src/main/java/dk/cngroup/lentils/service/UserService.java

```diff
@@ -0,0 +1,29 @@
+package dk.cngroup.lentils.service;
+
+import dk.cngroup.lentils.entity.User;
+import dk.cngroup.lentils.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class UserService {
+
+    public static final String ROLE = "ORGANIZER";
```

Comment:
Když se na to tak dívám po druhé, tahle konstanta by měla jít někdo do rolí. Asi do entity bych to dal. jako public konstantu.